### DISCUSSION TITLE
Connects From specific ISPs/PCs only

### DISCUSSION THREAD

**Discussion by MotazHakim** [pivpn/pivpn#1648] [Support]
Hello

I'm having a weird issue, and I believe it's not from PiVPN; it's from my PC or the ISP itself. 

So, I've 2 VPS (1 on Oracle Cloud - 1 DigitalOcean)
and I've installed PiVPN and used all default configurations. Only choose OPENVPN for client and for DNS I choose Google.
I've created a client file and opened the needed ports ( 80 / 443 / 1194 )

I tried the same client on 3 PCs
- every PC is using a different ISP
- every PC with [ Windows 10 / OpenVPN Client v3.3.6 (2752) ]

So 2 PCs are giving connection timeout; here's the [log ](https://pastebin.com/eZePrbRX) for them

and 1 PC is Connected and working like charm; here's the [Log ](https://pastebin.com/d3XggHRr)for it

the 2 PCs that are not connecting, I've access to the router, i've disabled the firewall, and I've port forward 1194 ... with no luck

I've even tried the same client on my iPhone, and it didn't work on Wifi or even 4G (another different ISP)

and what makes this issue weirder is if I removed PIVPN, then installed OpenVPN Access Server (the original source that allows 2 free clients) with all default configurations and port, and then created a client and tested it, IT WORKS ON EVERY PC AND PHONE !!!!!!!

### So, we can't say the ISP is blocking port 1194 because it works if I'm using OpenVPN Access server !! but not with PiVPN !!!

and I've used the command `tcpdump -n -i eth0 udp port 1194` to capture network packets and when i test it with the PC that is not working and tried to connect, it gives output like in this [screenshot ](https://images2.imgbox.com/41/a5/UfmTtgxm_o.jpg) but the connection never complete.

here's the server.conf content:

```
dev tun
proto udp
port 1194
ca /etc/openvpn/easy-rsa/pki/ca.crt
cert /etc/openvpn/easy-rsa/pki/issued/ubuntu_d044142c-49b9-4369-a88e-7576aaf6284d.crt
key /etc/openvpn/easy-rsa/pki/private/ubuntu_d044142c-49b9-4369-a88e-7576aaf6284d.key
dh none
ecdh-curve prime256v1
topology subnet
server 10.35.47.0 255.255.255.0
# Set your primary domain name server address for clients
push "dhcp-option DNS 8.8.8.8"
push "dhcp-option DNS 8.8.4.4"
# Prevent DNS leaks on Windows
push "block-outside-dns"
# Override the Client default gateway by using 0.0.0.0/1 and
# 128.0.0.0/1 rather than 0.0.0.0/0. This has the benefit of
# overriding but not wiping out the original default gateway.
push "redirect-gateway def1"
client-to-client
client-config-dir /etc/openvpn/ccd
keepalive 15 120
remote-cert-tls client
tls-version-min 1.2
 tls-crypt /etc/openvpn/easy-rsa/pki/ta.key
cipher AES-256-CBC
auth SHA256
user openvpn
group openvpn
persist-key
persist-tun
crl-verify /etc/openvpn/crl.pem
status /var/log/openvpn-status.log 20
status-version 3
syslog
verb 3
#DuplicateCNs allow access control on a less-granular, per user basis.
#Remove # if you will manage access by user instead of device. 
#duplicate-cn
# Generated for use by PiVPN.io
```

here's the client.ovpn content
```
client
dev tun
proto udp
remote 174.138.9.8 1194
resolv-retry infinite
nobind
remote-cert-tls server
tls-version-min 1.2
verify-x509-name ubuntu_d044142c-49b9-4369-a88e-7576aaf6284d name
cipher AES-256-CBC
auth SHA256
auth-nocache
verb 3
<ca>
-----BEGIN CERTIFICATE-----
I've removed the rest of cert content here
```
So, what's the issue here !!!!!!! because I'm going crazy right now!!

**Comment by TinCanTech** [comment]
> I'm going crazy right now!!

Don't go crazy, get mad: https://forums.openvpn.net/viewtopic.php?t=22603#p68963
